.github/workflows/lib-ms.yml

```diff
@@ -60,3 +60,42 @@ jobs:
         with:
           files: servers/lib/coverage/clover.xml
           flags: lib-microservice-tests
+
+  publish-package:
+
+    if: | 
+      github.event_name == 'push' &&
+      (startsWith(github.ref, 'refs/heads/feature/') || startsWith(github.ref, 'refs/heads/release-v'))
```

Comment:
please provide link to your github action logs which publish package from a branch starting with `release-v*`.

Reply:
https://github.com/lindot11/DTaaS/actions/runs/6798405636